core/chains/evm/forwarders/forwarder_manager.go

```diff
@@ -255,7 +255,6 @@ func (f *FwdMgr) runLoop() {
 				[]common.Hash{authChangedTopic},
 				addrs,
 				evmlogpoller.Confirmations(f.cfg.FinalityDepth()),
-				pg.WithParentCtx(f.ctx),
```

Comment:
Shouldn't all of these cases dropping `WithParentCtx` be passing the context as the first param instead?

Reply:
Nice catch, just added context to all of them.